playbooks/clouds.yaml.j2

```diff
@@ -119,3 +119,12 @@ clouds:
       application_credential_id: "{{ clouds_conf.wavestack_ac_id }}"
       application_credential_secret: "{{ clouds_conf.wavestack_ac_secret }}"
     auth_type: "v3applicationcredential"
+  cc-rrze:
+    region_name: "DE-ERL"
+    interface: "public"
+    identity_api_version: 3
+    auth_type: "v3applicationcredential"
+    auth:
+      auth_url: AUTH_URL
```

Comment:
This looks wrong.

Reply:
fixed with 8c50ef4c968c55ab3c0564bd4482c7b992c8deb7